sql/2024/fonts/development/fonts_compiler.sql

```diff
@@ -0,0 +1,48 @@
+-- Section: Design
+-- Question: Which compilers are used?
+
+CREATE TEMPORARY FUNCTION COMPILER(version STRING) AS (
+  CASE
+    WHEN REGEXP_CONTAINS(version, r'(?i)(Core \d|PS \d|hotconv|makeotf)') THEN 'Adobe Font Development Kit for OpenType'
+    WHEN REGEXP_CONTAINS(version, r'(?i)FontCreator') THEN 'FontCreator'
+    WHEN REGEXP_CONTAINS(version, r'(?i)Fontself') THEN 'Fontself Maker'
+    WHEN REGEXP_CONTAINS(version, r'(?i)ttfautohint') THEN 'FreeType'
```

Comment:
I don't think we should attribute ttfautohint to FreeType, as neither are font compilers and the actual "compiling" has been done by one of the other tools.

Reply:
Hm, that is true. Removed.